content/en/open-positions/backend-engineer.md

```diff
@@ -1,45 +1,53 @@
 ---
 title: Senior Backend Engineer (Remote)
-description: Come help us tighten up our back end. Not in the gym sort of way, but in the engineering / development sort of way.
 position: 2
 category: Open Positions
 ---
 
 ## What You Will Do
 
-We currently have a marketplace that connects buyers and sellers of adtech data, and we are building on this success 
-to make it possible to transact any other kinds of data.
-
-As a result, here are the kinds of projects you will likely work on in the foreseable future:
-- Make our core systems generic/agnostic to the actual data types that are being ingested, transacted and delivered. 
-  For instance, we are building Iceberg-backed data puddles, schema/validation tools, making the transaction process more generic, etc.
-- Improve the monitoring and reliability of our systems with the ever-increasing amount of data being handled (over 500GB per day)
-- Extract abstractions, modularize the code and improve the tests.
-- Enhance the API we expose to partners and increasingly rely on it for internal use, in order to "eat our own dogfood".
-
+We have built a marketplace that allows buyers and sellers to transact any kind of data.
+
+Here are the kinds of projects you will likely work on:
+
+- Add capabilities to our query execution and data classification engines to make it possible to fulfill any buyer
```

Comment:
This needs another set of eyes. I think I focused too much on the the query part of the product and not enough on the marketplace part, taking any and all feedback (e.g. suggestions for what too add, what to remove, etc.)